### PR TITLE
Document the user personal settings

### LINKED ISSUE #308: User Manual: describe settings : section : personal
If you look on https://doc.owncloud.org/server/latest/user_manual/, there is no part describing the personal settings section. There are many parts that are worth describing it especially for first time users or users who look for a particular answer in this section as they see items in the browser and want to know what it is about.

The left is the doc, the right is the personal settings page.

image

### PR DESCRIPTION
**Pull request by settermjd**
This change documents the personal settings section of the user manual.
It doesn't go into extensive depth, but sufficiently so such that the
various settings should be well enough covered.

This fixes #308.

### CODE REVIEW

File: modules/user_manual/pages/personal_settings/general.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
+= General Settings
+
```
Comment (mmattel) [inline modules/user_manual/pages/personal_settings/general.adoc]: no `:toc: right` ?

File: modules/user_manual/pages/personal_settings/general.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+In the general settings, you can do the following:
+
+* Set:
+** Your profile picture
+** xref:set-the-language[The language for your user account]
+* Update your:
```
Comment (mmattel) [inline modules/user_manual/pages/personal_settings/general.adoc]: What about, proposal: `Update your` --> `Update personal data`

File: modules/user_manual/pages/personal_settings/index.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
+= Personal Settings
+
```
Comment (mmattel) [inline modules/user_manual/pages/personal_settings/index.adoc]: no `:toc: right` ?

File: modules/user_manual/pages/personal_settings/security.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
+= Security
+:cors-url: https://en.wikipedia.org/wiki/Cross-origin_resource_sharing
```
Comment (mmattel) [inline modules/user_manual/pages/personal_settings/security.adoc]: no `:toc: right` ?

**Comment by settermjd** [comment]
Thanks for the feedback, @mmattel. All implemented.

**Comment by mmattel** [comment]
Backporting to all branches ?

**Pull request merged**